app/MMB/src/navigation/AddressPickerScreen.js

```diff
@@ -95,23 +102,41 @@ export class AddressPickerScreen extends React.Component<Props, State> {
     }
   });
 
+  setLocationToCurrentLocation = () => {
+    const location = this.getCoordinate();
+    if (location != null) {
+      this.props.navigation.state.params.setLocation(location);
+    }
+  };
+
   render() {
+    const searchText =
+      idx(this.props.navigation, _ => _.state.params.searchText) || '';
+    const region = idx(this.props.navigation, _ => _.state.params.region);
     return (
-      <ScrollView>
+      <ScrollView keyboardShouldPersistTaps="handled">
         {this.state.formattedAddress != '' && (
```

Comment:
I know it is not part of this PR, but this should probably have been `!== ''`

Reply:
Absolutely, will fix it.